TestingScripts/CommonTasks/Components.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
This is the json file I was speaking of . I can't add copyright comments here because comments are not allowed in .json files. Is this fine?

Reply:
yes